scripts/parse_data_from_cldr.py

```diff
@@ -0,0 +1,151 @@
+import os
```

Comment:
this script is not only parsing the data from CLDR, but also merging it with the `supplementary_translation_data`, so I would rename this script to just: `parse_numeric_data`, `merge_numeric_data` or something similar.